python_modules/dagster-webserver/dagster_webserver/webserver.py

```diff
@@ -234,27 +224,48 @@ def index_html_endpoint(self, request: Request):
                 "make rebuild_ui" in the project root.
                 """)
 
-    def root_static_file_routes(self) -> List[Route]:
```

Comment:
Changes in this file are probably of interest to @alangenfeld 

- Use os.walk to walk the build directory and use FileResponse to server all assets except for javascript files which go through a new route which replaces `BUILDTIME_ASSETPREFIX_REPLACE_ME` to preserve --path-prefix behavior 
- This eliminates the need to maintain `ROOT_ADDRESS_STATIC_RESOURCES`
- Which also makes robots.txt exposed (it wasn't in the list before).